guides/common/modules/proc_enabling-dependency-solving-per-content-view.adoc

```diff
@@ -1,8 +1,8 @@
-[id="enabling-dependency-solving-globally_{context}"]
-= Enabling Dependency Solving Globally
+[id="enabling-dependency-solving-per-content-view_{context}"]
+= Enabling Dependency Solving Per Content View
 
 You can enable dependency solving for a Content View or by default for your whole {Project} environment.
```

Comment:
This is no longer true. You can only enable it for individual content views.

Reply:
"no longer": Is this true for Foreman 3.7 all the way down to Foreman 3.1? @jeremylenz

Reply:
Good point! The setting removal isn't until Katello 4.10/Satellite 6.15. It would still be available in older versions that offer the old content view UI to create content views, but I don't remember when that was discontinued. 

Regardless, I think it's still coherent if we just change the sentence to apply to individual content views and don't mention the global setting.